doc/rtd/explanation/about-cloud-config.rst

```diff
@@ -0,0 +1,85 @@
+.. _about-cloud-config:
+
+About the cloud-config file
+***************************
+
+The ``#cloud-config`` file is a type of user data that cloud-init can consume
+to automatically set up various aspects of the system.
+
+It overrides all other types of user data, so if there is a conflict between
+the config you specify in other :ref:`user data formats <user_data_formats>`,
+the contents of the ``#cloud-config`` file will take precendence.
+
+Note that networks are not configurable via the ``#cloud-config`` file because
+:ref:`network configuration <network_config>` occurs before user data is
```

Comment:
This isn't really true. The main reason is that network configuration comes from the cloud and applying a different configuration could make the instance unreachable.

In most cases we retrieve the network config at the same time as the cloud-config and we *could* extract network-config at that time if we wanted to. We've floating the idea of supporting it a number of times.